search_params.py

```diff
@@ -14,9 +14,9 @@
 from pytorch_lightning.utilities.parsing import AttributeDict
 from ray import tune
 
-from libmultilabel import data_utils
 from libmultilabel import networks
```

Comment:
Shouldn't `networks/` be moved into `nn/`?

Reply:
Yes!! They should be in `nn/`.